### ISSUE TITLE
Request to add the Desikan-Killiany-Tourville human brain labeling protocol

### ISSUE THREAD

**Issue by binarybottle** [SciCrunch/NIF-Ontology#118]
It would be really nice to add the Desikan-Killiany-Tourville brain labeling protocol that is now used by default by the Mindboggle (http://mindboggle.info) and FreeSurfer software packages.

Here is a list of the labels:
http://mindboggle.readthedocs.io/en/latest/labels.html

Here is the primary reference article:
[101 labeled brain images and a consistent human cortical labeling protocol](http://www.frontiersin.org/Brain_Imaging_Methods/10.3389/fnins.2012.00171/full)
Arno Klein, Jason Tourville. Frontiers in Brain Imaging Methods. 6:171.
http://dx.doi.org/10.3389/fnins.2012.00171

**Comment by tgbugs** [comment]
Thank you for pointing me to this. I will include this the next time that I generate our [parcellation schemes](https://github.com/tgbugs/pyontutils/blob/master/pyontutils/parcellation.py). I'll ping you again if I have more questions/updates. Best!

**Comment by tgbugs** [comment]
@satra @binarybottle cc @jgrethe

I think that I have interpreted what is going on in [mindboggle.mio.DKTprotocol](https://github.com/nipy/mindboggle/blob/master/mindboggle/mio/labels.py) correctly. I have produced two files, one for the DKT regions and sulci and one for the mindboggle labels (i.e. the set from DKTprotocol.label_numbers and DKTprotocol.label_names). The source file that does the transformation is https://github.com/tgbugs/pyontutils/blob/master/pyontutils/parc_mndbgl.py.

A first pass at the output can be seen on the parcellation branch in [dkt-labels.ttl](url) and [mndbgl-labels.ttl](url). The artifacts can be seen [here](url) and [here](url) in parcellation-artifacts.ttl.

I do not know whether you want to deal with, nor how to deal with, representing the pair lists that define the sulci in the ontology. I'm guessing that it is probably best to leave that to mindboggle?

Please let me know it seems like I am on the right track. Thanks!

**Comment by satra** [comment]
@tgbugs - the labels change from DKT to DTK near the sulcal transition in the ttl file.

regarding the label themselves. the DKT protocol is exactly the same protocol in mindboggle and freesurfer. so in some sense they will share these labels and any software that uses the freesurfer or mindboggle atlases.

regarding how to turn the sulci into rdf relations we will have to introduce a specific notion of bounded by to associate sulci to the labels. but this starts getting into the weed of anatomical provenance for any parcel. hopefully that's something that can be standardized into a set of concepts across atlas creation.

**Comment by tgbugs** [comment]
@satra 
I'm working on a notion of landmarks for naming in parcellation schemes. Right now it is only developed for point based landmarks, but it should generalize to curves on a surface or 1d curves in a 3d space. The 'unbounded parcellation region' rightmost in the image in https://github.com/SciCrunch/NIF-Ontology/blob/master/docs/brain-regions.org#tables, is equivalent I think. If we classify the sulci labels as being attached landmarks of that sort can we safely leave the specification of their contour to the implementations?

**Comment by satra** [comment]
@tgbugs - will have to think about this one a bit. over the years, we have defined sulci using various methods. almost all of them end up equivalent to having neighboring landmarks, but not always. and those landmarks are not always regions. in mindboggle, because we rely on the DKT atlas, we can use it in a way to find the central sulcus.

however, see: https://www.coursera.org/learn/medical-neuroscience/lecture/wJ4kL/finding-the-central-sulcus for an alternate method.

i think key question is how methods like this get operationalized ? and perhaps more importantly how do two or more different approaches to getting at the same concept get treated equivalently or result in differences.

**Comment by tgbugs** [comment]
I agree about the importance of how a term is operationalized. I don't think we want to full operationaliziation in the ontology, but we do want to provide terms that experts can say do or do not map to some operationalization.

At the end of the day we want to be able to determine whether two concepts are scientifically equivalent. Hopefully it shouldn't matter which method we use to identify a sulculs, if it is a real thing there should be more than one way, but in order to demonstrate that scientifically we will want/need ontology terms that take the methodology into account, so that the provenance of the identification can be explicit, so that, for example, we can demonstrate the equivalence of a gross anatomically identified central sulcus and a mindboggle identified central sulcus.

I think that the goal for the ontology should probably be to link to a definition of the operationalization if one exists. So for the DKT sulci that would be the labels files or similar. Does this seem reasonable?

**Comment by satra** [comment]
@tgbugs - sounds reasonable

**Comment by tgbugs** [comment]
@satra cc @jgrethe 

I [fixed the k/t switch](https://github.com/SciCrunch/NIF-Ontology/commit/c57d73f19f43ed183f3a1d55c70a788247faa718#diff-e6f281ef76c06766bd308dcaa50e01a4) and have switched to [source](https://github.com/SciCrunch/NIF-Ontology/blob/parcellation/ttl/generated/parcellation-artifacts.ttl#L361) the FreeSurfer ids directly from their git repo. They are now in [freesurfer-labels.ttl](https://github.com/SciCrunch/NIF-Ontology/blob/parcellation/ttl/generated/parcellation/freesurfer-labels.ttl).

Two questions:

1. All of the naming is provisional, at the moment. Do you know if there are commonly used abbreviations for the freesurfer labels as a whole? Also any other suggestions with regard to naming for this?
2. Since I am now source most of the labels from FreeSurfer directly does it still make sense to set up DKT under the mindboggle organization, (given that, for the moment, DKT is the only terminology associated with the mindboggle org)? I imagine that you would be one of the admins on that when we get that functionality implemented.
